Config.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+
+
+class Config:
+    def __init__(self, config_file_path="config.txt"):
+        print('sys.path', sys.path)
+        print('current path of Config', config_file_path)
+        for path in sys.path:
+            if path.endswith('/BluePen'):
```

Comment:
What is `BluePen`? This is looking for a config file at a certain path but neither @aida-coughlan and I have this file on our machines. What is the plan for sharing this and ensuring it exists within our ec2 instance as well?